cf_remote/ssh.py

```diff
@@ -46,6 +46,8 @@ def put(self, src, hide=False):
 class Connection:
     def __init__(self, host, user, connect_kwargs=None):
         self.ssh_host = host
+        if ":" in host:
+            port = host.split(":")[-1]
```

Comment:
This will break with the first IPv6 address someone will try to use.

Reply:
sure thing, will refactor.

Reply:
ok, closes thing I can find is https://datatracker.ietf.org/doc/html/rfc3986 where they say 

```
   authority     = [ userinfo "@" ] host [ ":" port ]
   userinfo      = *( unreserved / pct-encoded / sub-delims / ":" )
   host          = IP-literal / IPv4address / reg-name
   port          = *DIGIT

   IP-literal    = "[" ( IPv6address / IPvFuture  ) "]"
```

And so an ipv6 address including a port would be [ipv6-addr]:port. I will go with that.

Reply:
https://www.ietf.org/rfc/rfc2732.txt mentions the [ipv6] format for URLs as well.

Reply:
https://docs.python.org/3/library/urllib.parse.html references lots of RFCs and handles this case. I will use it partially by prefixing `ssh://` to the host parameter somewhere.

Reply:
@vpodzime I will fix this also in aramid here: https://github.com/cfengine/cf-remote/blob/master/cf_remote/aramid.py#L194-L206

Reply:
Great, thanks!